datareduction/datareducer.cpp

```diff
@@ -79,10 +79,10 @@ void initializeDataReducers(DataReducer * outputReducer, DataReducer * diagnosti
                   }
                }
                return retval;
-	 }
-	 ));
-         outputReducer->addMetadata(outputReducer->size()-1,"T","$\\mathrm{T}$","$B$","1.0");
-         continue;
+            }
+         ));
+         outputReducer->addMetadata(outputReducer->size()-1,"T","$\\mathrm{T}$","$B_\\mathrm{bg,fg}$","1.0");
```

Comment:
This one is still all B, not just bgb. Feel free to add the fg descriptor, but the bg,fg pair would go to line 116.